include/muu/impl/std_type_traits.h

```diff
@@ -8,7 +8,9 @@
 #include "../preprocessor.h"
 
 MUU_DISABLE_WARNINGS;
-#include <type_traits>
+#if __has_include (<type_traits>)
+	#include <type_traits>
+#endif
```

Comment:
```suggestion
#include <type_traits>
```

This is unnecessary. Muu targets C++17 at a minimum; it's basically impossible for any C++17 standard library to lack `<type_traits>` since most of the library is built on it.